lib/app/widgets/form.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:schulcloud/app/app.dart';
+import '../utils.dart';
+
+Future<bool> showDiscardChangesDialog(BuildContext context) {
```

Comment:
I think it would be cleaner to make this an extension method on `BuildContext`. That makes it more intuitive to use and we're sure that `context != null`.